Vadim/assets/index/css/index.css

```
@@ -0,0 +1,676 @@
+}
+
+.hide {
+    display: none;
+}
+
+.bold {
+    font-weight: 700;
+}
+
+.black-inst {
+    color: #262626;
+}
+
+.border-hide {
+    border: none;
+}
+
+.border-inst {
+    border: 1px solid #DBDBDB;
+    border-radius: 3px;
+}
+/*==================================*/
+
+
+.header {
+    background: #fff;
+    z-index: 999;
+    position: fixed;
+    left: 50%;
+    transform: translate(-50%, -0%);
+    height: 53px;
+    border-bottom: solid 1px #DBDBDB;
+}
+
+.header-wrapper {
+    max-width: 975px;
+    padding: 0 20px;
+    justify-content: space-between;
+    height: 53px;
+}
+
+.header-aside {
+    font-weight: 700;
+    position: relative;
+    justify-content: center;
+    align-items: center;
+    font-size: 19px;
+}
+
+.header_aside-icon_protection {
+    font-size: 12px;
+    position: absolute;
+    left: -30%;
+    bottom: 3px;
+}
+
+.header_aside-btn_arrow {
+    color: #262626;
+    background: none;
+    position: absolute;
+    right: -35%;
+    top: 26%;
+}
+
+.header-bar {
+    position: relative;
+}
+
+.header-search {
+    max-width: 192px;
+    height: 29px;
+    color: #8F8F8F;
+    text-align: center;
+    background: #fafafa;
+    padding: 0 35px;
+}
+
+.header-search::placeholder {
+    color: #8F8F8F;
+  }
+
+.header-search-icon {
+    background: none;
+    position: absolute;
+    top: 50%;
+    left: 4%;
+    transform: translate(-5%, -50%);
+}
+
+.header-buttons {
+    position: relative;
+    justify-content: space-between;
+    max-width: 198px;
+    height: 22px;
+} 
+
+.header-buttons button {
+    background: none;
+}
+
+.header-button {
+    position: relative;
+}
+
+.link-circle_icon {
+    height: 22px;
+    width: 22px;
+    border-radius: 50%;
+    background: #4094EF;
+}
+
+.link-circle_icon:hover {
+    background: #DC565B;
+}
+
+.header_button-notification {
+    position: absolute;
+    width: 22px;
+    height: 22px;
+    color: #fff;
+    background: #DC565B;
+    border-radius: 50%;
+    font-size: 11px;
+    top: -47%;
+    right: -19%;
+}
+
+.button_profile-menu {
+    background: #fff;
+    z-index: 999;
+    position: fixed;
+    width: 207px;
+    height: 210px;
+    top: 50px;
+    border-radius: 4px;
+    box-shadow: 0 0 5px 4px rgba(var(--jb7,0,0,0),.0975);
+}
+
+.profile_menu-rhombus {
+    background: #fff;
+    position: absolute;
+    width: 15px;
+    height: 15px;
+    right: 9%;
+    top: -3%;
+    transform: rotate(45deg);
+    border-radius: 4px;
+    box-shadow: 84px -57px 18px -6px rgba(0, 0, 0, 0.6);
+}
+
+.profile_menu-list {
+    font-size: 13px;
+    color: #545454;
+}
+
+.profile_menu_list-item {
+    display: flex;
+    align-items: center;
+    padding: 13px 10px;
+}
+
+.profile_menu_list-item:hover {
+    color: #4094EF;
+}
+
+.profile_menu_list-item:last-child {
+    border-top: 1px solid #DBDBDB;
+    padding: 16px 0px;
+}
+
+.profile_menu_list-item p {
+    margin: 0 0 0 20px;
+}
+
+.profile_menu_list-item:last-child p {
+    margin: 0 0 0 10px;
```

```suggestion
    margin-left: 10px;
```

И так много где нужно поправить будет